### PR TITLE
initial picture handling in jimi output

### PR DESCRIPTION
**Pull request by petrjasek**
SDCP-228

### CODE REVIEW

File: server/cp/orangelogic.py [hunk 1]
```diff
@@ -230,12 +244,86 @@ def fetch(self, guid):
...
         return item
 
 
+def _parse_binary(item):
+    binary = app.media.get(item['renditions']['original']['media'])
+    iptc = get_meta_iptc(binary)
+    binary.seek(0)
+    xmp = parse_xmp(binary)
+
+    if not iptc:
```
Comment (ride90) [inline server/cp/orangelogic.py:264]: we can put it before 
```
binary.seek(0)
xmp = parse_xmp(binary)
```

File: server/cp/output/formatter/jimi.py [hunk 2]
```diff
@@ -87,34 +102,57 @@ def _format_item(self, root, item, pub_seq_num, service, services):
...
+        if item['type'] == 'text' and item.get('body_html'):
+            content.find('DirectoryText').text = format_maxlength(
+                get_text(item['body_html'], 'html', lf_on_block=False).replace('\n', ' '),
+                200)
+            word_count = str(
+                item['word_count'] if item.get('word_count') else get_word_count(item['body_html'])
+            )
+            etree.SubElement(content, 'Length').text = word_count
+            etree.SubElement(content, 'WordCount').text = word_count
+            etree.SubElement(content, 'BreakWordCount').text = word_count
```
Comment (ride90) [inline server/cp/output/formatter/jimi.py:127]: it is the same as a `WordCount`, right?

File: server/dev-requirements.txt [hunk 3]
```diff
@@ -5,3 +5,4 @@ nose
 pep8
 httmock
 wooper
+requests_mock
```
Comment (ride90) [inline server/dev-requirements.txt:8]: I think this one is already in dev-requirements in the core

Comment (ride90) [reply server/dev-requirements.txt:8]: ahh, makes total sense to have it here

File: server/cp/ingest/parser/ap.py [hunk 4]
```diff
@@ -529,6 +557,57 @@ def _parse_category(self, data, item):
                     item['anpa_category'].append(
                         {'name': cat['name'], 'qcode': cat['qcode']},
                     )
+                    break
+
+    def _parse_picture_category(self, data, item):
+        for subj in data['item'].get('subject', []):
```
[the remark this answers is not in the document]
Comment (petrjasek) [reply server/cp/ingest/parser/ap.py:562]: it's always there

File: client/extensions/upload-iptc/src/extension.ts [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+                        if (created && created !== '') {
+                            item.firstcreated = created;
+                        }
+
+                        if (release) {
+                            item.extra.DateRelease = release;
+                        }
+                    }
+
+                    console.debug('iptc', data, item);
```
[the remark this answers is not in the document]
Comment (petrjasek) [reply client/extensions/upload-iptc/src/extension.ts:82]: this one I might still use for debugging

File: server/cp/output/formatter/jimi.py (hunk 2 above)
Comment (petrjasek) [reply server/cp/output/formatter/jimi.py:127]: right

**Pull request merged**